docs/variants/qemu_q35/building-manual.md

```diff
@@ -0,0 +1,48 @@
+#Dasharo (UEFI) v0.1.0 for QEMU Q35 - Building Manual
+
+In order to build the `Dasharo (UEFI) OVMF firmware image for QEMU Q35`, please follow the given steps to build the Dasharo firmware image from the EDK2 repository:
+
+1. Clone the official Dasharo EDK2 repository to your local instance with git or downloading the source code from github.
+
+    ```bash
+    git clone https://github.com/Dasharo/edk2.git 
+    ```
+       
+2. Once it is done, please follow the below instructions, in order to prepare your environment for the building of the OVMF image.
+
+	
+	* Change the directory to Dasharo EDK2 repository locally - 
+	
+	```bash
+	cd Dasharo/edk2
+	```
+	
+	
+	
+	* Setup the environment variables with the following command - 
+	
+	```bash
+	source edksetup.sh
+	```
+								       
+	* Update the submodules in order get the latest dependencies and avoid any build process errors while building the OVMF image. 
+	
+	```bash
+	git submodules update --init
+	```
+									       
```

Comment:
`make -C BaseTools` has to be invoked too, otherwise `build` command will not be recognized.

Reply:
Noted. Will work on it.